<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>个人工作与任务管理系统</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <widget class="QGroupBox" name="groupBoxStatistic">
      <property name="title">
       <string>任务统计</string>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <item>
        <widget class="QLabel" name="labelTotal">
         <property name="text">
          <string>总任务数：0</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="labelCompleted">
         <property name="text">
          <string>已完成任务数：0</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="labelOverdue">
         <property name="text">
          <string>逾期未完成任务数：0</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QGroupBox" name="groupBoxFilter">
      <property name="title">
       <string>任务筛选</string>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string>分类：</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="comboCategoryFilter"/>
       </item>
       <item>
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>优先级：</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="comboPriorityFilter"/>
       </item>
       <item>
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>状态：</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="comboStatusFilter"/>
       </item>
       <item>
        <widget class="QPushButton" name="btnRefreshFilter">
         <property name="text">
          <string>刷新筛选</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QTableView" name="tableViewTasks"/>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QPushButton" name="btnAdd">
        <property name="text">
         <string>添加任务</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnEdit">
        <property name="text">
         <string>编辑任务</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnDelete">
        <property name="text">
         <string>删除任务</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="btnExportPdf">
        <property name="text">
         <string>导出PDF</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnExportCsv">
        <property name="text">
         <string>导出CSV</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
